athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/optimizations/ComplexExpressionPushdownSubType.java

```diff
@@ -0,0 +1,42 @@
+/*-
+ * #%L
+ * Amazon Athena Query Federation SDK
+ * %%
+ * Copyright (C) 2019 - 2022 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connector.lambda.metadata.optimizations;
+
+public enum ComplexExpressionPushdownSubType
+        implements PushdownSubTypes
+{
+    NONE("none"),
+    SUPPORTS_CONSTANT_EXPRESSION_PUSHDOWN("supports_constant_expression_pushdown"),
+    SUPPROTS_FUNCTION_CALL_EXPRESSION_PUSHDOWN("supports_function_call_expression_pushdown"),
+    SUPPOROTS_VARIABLE_EXPRESSIONS_PUSHDOWN("supports_variable_expression_pushdown");
```

Comment:
Are these typos? Did you mean "SUPPORTS"?

Reply:
great catch! proof of error in my mac butterfly keyboard. will update

note - this won't actually change the serde, we can see the string "supports_variable_expression_pushdown" has no typo, and that's what is sent over the wire.